csharp/src/Ice/SlicSocket.cs

```diff
@@ -23,8 +23,9 @@ public override TimeSpan IdleTimeout
 
         internal int BidirectionalStreamCount;
         internal AsyncSemaphore? BidirectionalStreamSemaphore;
+        internal int MaxBidirectionalStreams;
```

Comment:
In general for non private fields / properties, we prefer properties.

Reply:
I'll fix this in another PR.